packages/patternfly-react/src/components/LoginPage/assets/img/logos/index.js

```diff
@@ -0,0 +1,27 @@
+import dropbox from './dropbox-logo.svg';
```

Comment:
we do not need to duplicate these logos here in patternfly-react. They can be consumed/re-exported directly from Patternfly since we are directly dependent on PF.

That being said, I think we can change this to : 
```
import dropbox from 'patternfly/dist/img/dropbox-logo.svg';
import facebook from 'patternfly/dist/img/facebook-logo.svg';
import fedora from 'patternfly/dist/img/fedora-logo.png';
import git from 'patternfly/dist/img/git-logo.svg';
import github from 'patternfly/dist/img/github-logo.svg';
import google from 'patternfly/dist/img/google-logo.svg';
import instagram from 'patternfly/dist/img/instagram-logo.png';
import linkedin from 'patternfly/dist/img/linkedin-logo.svg';
import openID from 'patternfly/dist/img/open-id-logo.svg';
import skype from 'patternfly/dist/img/skype-logo.svg';
import stackExchange from 'patternfly/dist/img/stack-exchange-logo.svg';
import twitter from 'patternfly/dist/img/twitter-logo.svg';
```

and delete all of the new logo assets coming from PF...